### ISSUE TITLE
SVGs at 48 optical size

### ISSUE THREAD

**Issue by Markar** [marella/material-symbols#31]
The default optical size on [https://fonts.google.com/icons] is 24, but only 48 is available in @material-symbols/svg-x. I haven't found a way to change the optical size using styles yet. Is there another option for 24 optical size svgs? I saw the comment that additional sizes weren't included due to package limits, but it doesn't mention any potential workarounds.

**Comment by jgreffe** [comment]
Agree, same limits with https://github.com/marella/material-symbols/tree/main/font/400#readme

**Comment by senghuotlay** [comment]
same issue for me as well

**Comment by ChintanRajpara** [comment]
same issue for me as well

**Comment by egemon** [comment]
I was strugling to use it with MUI.
Here is my solution, hope it will help someone:
PS: obviously loader for SVG is need which transforms svg to react component (docs in this package)

```
import AddOutlined from '@material-symbols/svg-400/outlined/add.svg'
import CheckOutlined from '@material-symbols/svg-400/outlined/check.svg'
import CircleOutlined from '@material-symbols/svg-400/outlined/circle.svg'
import CloseOutlined from '@material-symbols/svg-400/outlined/close.svg'
import InfoOutlined from '@material-symbols/svg-400/outlined/info.svg'
import KeyboardArrowDownOutlined from '@material-symbols/svg-400/outlined/keyboard_arrow_down.svg'
import LogoutOutlined from '@material-symbols/svg-400/outlined/logout.svg'
import PersonOutlined from '@material-symbols/svg-400/outlined/person.svg'
import RemoveOutlined from '@material-symbols/svg-400/outlined/remove.svg'
import { SvgIcon, SvgIconProps } from '@mui/material'

const ICONS = {
  person: PersonOutlined,
  logout: LogoutOutlined,
  keyboard_arrow_down: KeyboardArrowDownOutlined,
  add: AddOutlined,
  remove: RemoveOutlined,
  close: CloseOutlined,
  check: CheckOutlined,
  circle: CircleOutlined,
  info: InfoOutlined,
} as const

type IconName = keyof typeof ICONS

interface BaseIconProps {
  icon: IconName
}

const BaseIcon = ({ icon }: BaseIconProps) => {
  const IconComponent = ICONS[icon]

  return <IconComponent />
}

export interface IconProps extends SvgIconProps, BaseIconProps {}

export const Icon = ({ icon, ...props }: IconProps) => (
  <SvgIcon viewBox="0 0 48 48" {...props}>
    <BaseIcon icon={icon} {...props} />
  </SvgIcon>
)
```

**Comment by jjjjonathan** [comment]
Same issue here. FWIW, I find it odd this package uses optical size 48 when the default is 24. The icons appear quite a bit thinner than I think was intended for general use at optical size 48.

**Comment by damisparks** [comment]
@egemon, is 48 the optical size you want? 
Are you trying to transform the SVG into a React component using the MUI API?

**Comment by damisparks** [comment]
> I was strugling to use it with MUI. Here is my solution, hope it will help someone: PS: obviously loader for SVG is need which transforms svg to react component (docs in this package)
> 
> ```
> import AddOutlined from '@material-symbols/svg-400/outlined/add.svg'
> import CheckOutlined from '@material-symbols/svg-400/outlined/check.svg'
> import CircleOutlined from '@material-symbols/svg-400/outlined/circle.svg'
> import CloseOutlined from '@material-symbols/svg-400/outlined/close.svg'
> import InfoOutlined from '@material-symbols/svg-400/outlined/info.svg'
> import KeyboardArrowDownOutlined from '@material-symbols/svg-400/outlined/keyboard_arrow_down.svg'
> import LogoutOutlined from '@material-symbols/svg-400/outlined/logout.svg'
> import PersonOutlined from '@material-symbols/svg-400/outlined/person.svg'
> import RemoveOutlined from '@material-symbols/svg-400/outlined/remove.svg'
> import { SvgIcon, SvgIconProps } from '@mui/material'
> 
> const ICONS = {
>   person: PersonOutlined,
>   logout: LogoutOutlined,
>   keyboard_arrow_down: KeyboardArrowDownOutlined,
>   add: AddOutlined,
>   remove: RemoveOutlined,
>   close: CloseOutlined,
>   check: CheckOutlined,
>   circle: CircleOutlined,
>   info: InfoOutlined,
> } as const
> 
> type IconName = keyof typeof ICONS
> 
> interface BaseIconProps {
>   icon: IconName
> }
> 
> const BaseIcon = ({ icon }: BaseIconProps) => {
>   const IconComponent = ICONS[icon]
> 
>   return <IconComponent />
> }
> 
> export interface IconProps extends SvgIconProps, BaseIconProps {}
> 
> export const Icon = ({ icon, ...props }: IconProps) => (
>   <SvgIcon viewBox="0 0 48 48" {...props}>
>     <BaseIcon icon={icon} {...props} />
>   </SvgIcon>
> )
> ```

@egemon This won't work since the initial SVG was created at the wrong optical size.

**Comment by WinsleyJ** [comment]
Same issue here!
